### ISSUE TITLE
AVT 1 - OverflowMenu w/ custom trigger DAP violations

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#4200]
## Enviroment
macOS 10.14.6
Chrome Version 77.0.3865.90
VoiceOver
## DAP 1.8.0.0

Description
Run DAP on the OverflowMenu w/ custom trigger Component and the following violation displays:
<img width="1069" alt="DAP violation Screen Shot" src="https://user-images.githubusercontent.com/21676914/66137363-95bb3180-e5c2-11e9-8186-2b1181becf14.png">
